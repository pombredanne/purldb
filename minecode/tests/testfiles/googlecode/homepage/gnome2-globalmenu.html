

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "projectName": "gnome2-globalmenu", "projectHomeUrl": "/p/gnome2-globalmenu", "relativeBaseUrl": "", "loggedInUserEmail": null, "profileUrl": null, "token": null, "domainName": null, "assetHostPath": "http://www.gstatic.com/codesite/ph"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-7056396-1'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 gnome2-globalmenu -
 
 
 Global Menu Bar for GNOME - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fgnome2-globalmenu%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fgnome2-globalmenu%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/gnome2-globalmenu">
 <a href="/p/gnome2-globalmenu/">
 
 
 <img src="/p/gnome2-globalmenu/logo?cct=1238106393"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/gnome2-globalmenu/"><span itemprop="name">gnome2-globalmenu</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/gnome2-globalmenu/"><span itemprop="description">Global Menu Bar for GNOME</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/gnome2-globalmenu/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/gnome2-globalmenu/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/gnome2-globalmenu/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/gnome2-globalmenu/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/gnome2-globalmenu/wiki/GitRepository?tm=4"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/gnome2-globalmenu/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/gnome2-globalmenu/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/gnome2-globalmenu/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.gnu.org/licenses/old-licenses/gpl-2.0.html" rel="nofollow">GNU GPL v2</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:gnome">gnome</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:globalmenu">globalmenu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:menu">menu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:menubar">menubar</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:libgnomenu">libgnomenu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:globalmenu">globalmenu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:gnomenu">gnomenu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:gtk2-aqd">gtk2-aqd</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:aqd">aqd</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:gtk">gtk</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:gtkmenubar">gtkmenubar</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:macmenu">macmenu</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:document-centric">document-centric</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/105371600056615308574/">rainwood...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/111767282021120992046/">pierre.s...@gmail.com</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">8 committers</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_filedownload-g16 goog-inline-block vt"></div>
 </div>
 <span><b>Downloads</b></span>
 </div>
 <ul class="pslist">
 
 <li class="psmeta nowrap">
 <a title="Download" href="//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.7.10.tar.bz2" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.7.10.tar.bz2', 'Featured']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.7.10.tar.bz2', 'Featured']);">
 </a>
 <a href="downloads/detail?name=gnome-globalmenu-0.7.10.tar.bz2"
 title="0.7.10 release">gnome-globalmenu-0.7.10.tar.bz2</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha4.tar.bz2" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha4.tar.bz2', 'Featured']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha4.tar.bz2', 'Featured']);">
 </a>
 <a href="downloads/detail?name=gnome-globalmenu-0.9.alpha4.tar.bz2"
 title="0.9 branch alpha4">gnome-globalmenu-0.9.alpha4.tar.bz2</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha5.tar.bz2" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha5.tar.bz2', 'Featured']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//gnome2-globalmenu.googlecode.com/files/gnome-globalmenu-0.9.alpha5.tar.bz2', 'Featured']);">
 </a>
 <a href="downloads/detail?name=gnome-globalmenu-0.9.alpha5.tar.bz2"
 title="fixes a deadlock and handling manager reboot correctly">gnome-globalmenu-0.9.alpha5.tar.bz2</a>
 </li>
 
 <li class="psmeta"><a href="downloads/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_paper_pencil-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Wiki pages</b></span>
 </div>
 <ul class="pslist nowrap">
 
 <li class="psmeta">
 <a href="wiki/AppGuideLines" title=""
 >AppGuideLines</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/DevelopmentHome" title="This page is the home page for development documents, ideas, drafts and other garbages."
 >DevelopmentHome</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/FAQ" title="Frequently Asked Questions"
 >FAQ</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/InstallGnome3" title="One-sentence summary of this page."
 >InstallGnome3</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/Installation" title="installation on various linux platforms"
 >Installation</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/InstallingonUbuntu" title="A step by step guide for Hardy/Intrepid/Jaunty"
 >InstallingonUbuntu</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/ReleaseNotes" title="Release Notes"
 >ReleaseNotes</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/Works" title=""
 >Works</a>
 </li>
 
 <li class="psmeta"><a href="w/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 <li class="psmeta"><b>Blogs</b></li>
 
 <li class="psmeta">
 <a href="http://rainwoodman.dreamhosters.com/wp" rel="nofollow">Yu</a>
 </li>
 
 <li class="psmeta">
 <a href="http://ayozone.org/tag/global-menu/" rel="nofollow">Ayoli&#39;s Tutorials</a>
 </li>
 
 <li class="psmeta">
 <a href="http://qiacat.blogspot.com/" rel="nofollow">Blog on the KDE port</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="http://bugzilla.gnome.org/show_bug.cgi?id=353076" rel="nofollow">Gnome Bug</a>
 </li>
 
 <li class="psmeta">
 <a href="https://launchpad.net/~globalmenu-team" rel="nofollow">Ubuntu Launchpad</a>
 </li>
 
 <li class="psmeta">
 <a href="http://www.ohloh.net/projects/11806?p=gnome-globalmenu" rel="nofollow">Ohloh</a>
 </li>
 
 <li class="psmeta">
 <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=411934" rel="nofollow">Firefox Bug</a>
 </li>
 
 <li class="psmeta">
 <a href="http://fedoraproject.org/wiki/Features/GlobalMenu" rel="nofollow">Fedora Feature Proposal</a>
 </li>
 
 <li class="psmeta">
 <a href="http://code.google.com/p/google-highly-open-participation-gnome/issues/detail?id=75&amp;colspec=ID%20Status%20ClaimedBy%20Summary" rel="nofollow">GHOP Testing Page</a>
 </li>
 
 <li class="psmeta">
 <a href="http://gnomefiles.org/app.php/Global_Menu" rel="nofollow">Gnome Files</a>
 </li>
 
 <li class="psmeta">
 <a href="http://gnome-globalmenu.googlecode.com" rel="nofollow">Project Alias</a>
 </li>
 
 <li class="psmeta">
 <a href="https://translations.edge.launchpad.net/globalmenu/trunk/+translations" rel="nofollow">Translate</a>
 </li>
 
 <li class="psmeta">
 <a href="http://ubuntuforums.org/showthread.php?t=241868" rel="nofollow">Ubuntu Forum Thread</a>
 </li>
 
 <li class="psmeta">
 <a href="http://carcosa.net/jason/software/obsolete/gnome-globalmenu/" rel="nofollow">A GTK 1.x GlobalMenu (obsolete)</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>Groups</b></li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/gnome-globalmenu" rel="nofollow">Discussion Group</a>
 </li>
 
 <li class="psgap"/>
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p><p align="center"> </p><h1><a name="Global_Menu_Bar_for_GNOME"></a>Global Menu Bar for GNOME<a href="#Global_Menu_Bar_for_GNOME" class="section_anchor"></a></h1><p></p> </p><p>Moving to GNOME 3. The new 0.9.x branch is based on GNOME 3. (checkout git branch gnome-3, will merge to master soon). </p><p><!--&lt;p align=&quot;center&quot;&gt; We need a Packager/PPA wizard to package !GlobalMenu for Karmic. If you think you have what it takes, &lt;a href=&quot;http://code.google.com/p/gnome2-globalmenu/issues/detail?id=524&amp;q=ppa&quot;&gt;contact us&lt;/a&gt;&lt;/p&gt;&lt;img src=&quot;http://gnome2-globalmenu.googlecode.com/issues/attachment?aid=-5081163315604217274&amp;name=48px-Preferences-desktop-locale.png&quot;/&gt;--> <p align="center"><a href="https://translations.edge.launchpad.net/globalmenu/trunk/+translations" rel="nofollow">28 new GlobalMenu strings to translate !</a>. It&#x27;s easy to translate, you can get started in 5 minutes and make a difference using Launchpad. </p> </p><p><!-- &lt;p align=&quot;center&quot;&gt;For all eye-candy lovers: [http://www.deviantart.com/download/122054947/Gloobus_Preview_0_03_2_by_JordiHP.gz Instant-preview all your files fullscreen without having to open any application with Gloobus!(DEB 32/64 bits)] </p><p>&lt;p align=&quot;center&quot;&gt;Great News! The !GlobalMenu PPA is now included as a choice in Ubuntu Tweak. You just have to get the latest version of Ubuntu Tweak and click on install !GlobalMenu. That&#x27;s a snap ! 
 &lt;/p&gt; &lt;p align=&quot;center&quot;&gt; So you want to improve the Global Menu Linux scene over the summer and get paid for it ? Google&#x27;s Summer of Code just gives you the chance to do that. Head to [http://code.google.com/p/gnome2-globalmenu/wiki/SummerOfCode2009 our list of ideas] or come up with your own. Just prepare and submit an application in [http://code.google.com/p/gnome2-globalmenu/issues/detail?id=331#c13 the relevant upstream organisation(s)] before April 3rd.Details at [http://socghop.appspot.com/program/home/google/gsoc2009] &lt;/p&gt;--> </p><p><!--&lt;p align=&quot;center&quot;&gt;&lt;wiki:video url=&quot;http://www.youtube.com/watch?v=YFM7_nDcYlc&quot;/&gt; --> <p align="center"> </p><p>Global Menu is the globally-shared menu bar of all applications launched in your desktop session. </p><p><p align="center"><img src="http://gnome2-globalmenu.googlecode.com/svn/wiki/GlobalMenuWindows.png"> </p><p>This project introduces document-oriented concepts into GNOME, and improves GNOME&#x27;s respect for Fitts&#x27;s law. Most GTK applications work just fine with Global Menu. See <a href="/p/gnome2-globalmenu/wiki/ApplicationCompatibility">ApplicationCompatibility</a> for a list of them. </p><p><p align="center"><a href="http://www.facebook.com/pages/GlobalMenu/266424888760" rel="nofollow"><img src="http://gnome2-globalmenu.googlecode.com/issues/attachment?aid=-3013907031449089010&amp;name=5u84f48n.gif&amp;inline=1"></img></a> </p> </p><p>There are several advantages by adopting a shared global menu bar in Gnome (thanks Matthew for organizing these):  <ul><li>It works better with narrow windows, because the width of the menus isn&#x27;t limited to the width of the window. (This is a problem for Gimp and Inkscape especially.) </li><li>It&#x27;s less confusing -- when two menu bars are visible on-screen at once, sometimes people choose the wrong one. </li><li>Global Menu is the first step to move toward a Document Centric Desktop Environment ( <a href="/p/gnome2-globalmenu/wiki/ThoughtsOnADocumentCentricGnome">ThoughtsOnADocumentCentricGnome</a> ) which is, according to us, a long-term trend in DEs. </li></ul></p><p><p align="center">Show your support for GlobalMenu by voting it up in Ubuntu Brainstorm <p align="center"><a href="http://brainstorm.ubuntu.com/idea/3136/" rel="nofollow"><img src="http://brainstorm.ubuntu.com/idea/3136/image/1/?filetype=.png" /></a> </p><h2><a name="Development"></a>Development<a href="#Development" class="section_anchor"></a></h2><blockquote>See: <a href="/p/gnome2-globalmenu/wiki/DevelopmentHome">DevelopmentHome</a> 
</blockquote><h2><a name="Installation"></a>Installation<a href="#Installation" class="section_anchor"></a></h2><blockquote>See: <a href="/p/gnome2-globalmenu/wiki/Installation">Installation</a> 
</blockquote><h2><a name="Links"></a>Links<a href="#Links" class="section_anchor"></a></h2><p>The Bug corresponding to the inclusion of a global menu in GNOME. It was opened by Aqd and has the old version of the patch. It keeps track of any progress, and of a future inclusion as an hidden option in GNOME.  <ul><li><a href="http://bugzilla.gnome.org/show_bug.cgi?id=353076" rel="nofollow">http://bugzilla.gnome.org/show_bug.cgi?id=353076</a> </li></ul></p><p>The Ubuntuforums.org thread started by Aqd to keep track of the development and to provide technical support. It is the major source of knowledge, tips and tricks, patches on the subject. It has been partly summarized in the Wiki but would need further synthetization. It is now the home of further development by rainwoodman. It is more than 150 pages long. <ul><li><a href="http://ubuntuforums.org/showthread.php?t=241868" rel="nofollow">http://ubuntuforums.org/showthread.php?t=241868</a> </li></ul></p><p>
<script src="//code-opensocial.googleusercontent.com/gadgets/js/rpc.js?c=1&amp;container=code"></script>
<script type="text/javascript">
function resizeIframeHandler(opt_height) {
  var elem = document.getElementById(this.f);
  if (!elem) return;
  if (!opt_height) {
    elem.style.height = undefined;
  }
  else {
    opt_height = Math.max(10, opt_height);
    elem.style.height = opt_height + 'px';
  }
}
gadgets.rpc.register("resize_iframe", resizeIframeHandler);

gadgets.rpc.register('set_title', function(title) {
 var elem = document.getElementById(this.f + '_title');
 if (elem) {
   elem.innerHTML = gadgets.util.escape(title);
 }
});
</script>
<h2 id="gadget0_title" class="gadget-title"></h2><iframe src="//code-opensocial.googleusercontent.com/gadgets/ifr?url=http%3A%2F%2Fwww.ohloh.net%2Fp%2F11806%2Fwidgets%2Fproject_basic_stats.xml&amp;parent=%2F%2Fcode.google.com%2Fhosting&amp;container=code" id="gadget0" name="gadget0" frameborder="1" height="220"></iframe><pre><span class="error">&lt;nobr&gt;</span></pre><h2 id="gadget1_title" class="gadget-title"></h2><iframe src="//code-opensocial.googleusercontent.com/gadgets/ifr?url=http%3A%2F%2Fwww.ohloh.net%2Fp%2F11806%2Fwidgets%2Fproject_cocomo.xml&amp;parent=%2F%2Fcode.google.com%2Fhosting&amp;container=code" id="gadget1" name="gadget1" frameborder="0" height="240"></iframe> </p><p><strong>THIS IS NON-WARRANTED SOFTWARE. USE AT YOUR OWN RISK. IT KILLS KITTENS AND COMPUTERS TOO</strong> </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

