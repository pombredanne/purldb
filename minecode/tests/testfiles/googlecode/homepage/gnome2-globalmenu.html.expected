{'content_license': None,
 'content_license_url': None,
 'description': u'Global Menu Bar for GNOME',
 'downloads_url': u'http://code.google.com/p/gnome2-globalmenu/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/gnome2-globalmenu/feeds',
 'home_url': u'http://code.google.com/p/gnome2-globalmenu/',
 'id': u'gnome2-globalmenu',
 'issues_url': u'http://code.google.com/p/gnome2-globalmenu/issues/list',
 'keywords': [],
 'labels': [u'aqd',
            u'document-centric',
            u'globalmenu',
            u'gnome',
            u'gnomenu',
            u'gtk',
            u'gtk2-aqd',
            u'gtkmenubar',
            u'libgnomenu',
            u'macmenu',
            u'menu',
            u'menubar'],
 'license': u'GNU GPL v2',
 'license_url': u'http://www.gnu.org/licenses/old-licenses/gpl-2.0.html',
 'name': u'gnome2-globalmenu',
 'peoples_url': u'http://code.google.com/p/gnome2-globalmenu/people/list',
 'vcs_url': u'http://code.google.com/p/gnome2-globalmenu/wiki/GitRepository?tm=4',
 'wiki_url': u'http://code.google.com/p/gnome2-globalmenu/w/list'}